static_files/css/style.css

```diff
@@ -65,61 +55,58 @@ ul {
 }
 
 .CathoLogo {
-
     align-self: center;
 }
 
-
-
 .VisibleMenu {
-
     color: #BCBF12;
     font-weight: bold;
     display: flex;
     align-items: center;
 }
 
-
 .VisibleMenu a {
-
     color: #BCBF12;
 }
 
 .SandwichButton {
```

Comment:
how about indicate to the user, that button is clickable?
Using `cursor: pointer`.